Microsoft/vsts-vscode
# Add delete menu to file explorer

In this PR, we've removed the automatic promoting of file deletions from candidate to pending changes.  This should address #241.

Tested scenarios:

1. At command line, delete a versioned file via filesystem, re-create file with same name, shown as EDIT, defaults to included
    - move between excluded and included groups, works?  yes
    - run "undo", works? yes
2. At command line, delete a versioned file via filesystem, file shows in excluded changes, works?  yes
    - move to included list (tf delete should get run on it), works?  yes
    - move to excluded list, behaves as expected, works?  yes
    - move back to included list (tf delete should NOT be run on it again), works?  yes
3. Use "Delete (TFVC)" menu item, item ends up in included list, works?  yes
    - move to excluded list, works?  yes
    - move back and forth between the two lists, works?  yes
4. Use "Delete (TFVC)" menu item, item ends up in included list, run undo, changes are undone?  yes
5. 'tf delete' does not show up on F1, works?  yes
